doc/source/whats_new.rst

```diff
@@ -12,6 +12,8 @@ Changelog
 
     - Add epochs browser to interactively view and manipulate epochs by `Jaakko Leppakangas`_
 
+    - Add ``plot_topo`` method to ``Evoked`` object which deprecates ``topo.plot_topo`` by `Jaakko Leppakangas`_
```

Comment:
The deprecated plot_topo is in topo module. I don't know how I can refer to it otherwise.